docs/Concepts/Protocol-Upgrades.md

```diff
@@ -1,26 +1,29 @@
-description: Protocol Upgrades      
+description: Protocol upgrades
 <!--- END of page meta data -->
 
-# Network upgrades in private networks 
+# Network upgrades in private networks
 
-Network upgrades are the mechanism for upgrading the Ethereum protocol. The point in time at 
-which the protocol upgrade is applied is the network upgrade. 
+Network upgrades are the mechanism for upgrading the Ethereum protocol. The point in time at which
+the protocol upgrade occurs is the network upgrade.
 
-For the Ethereum mainnet and public testnets (for example, Rinkeby) the milestone blocks are defined in 
-Hyperledger Besu. Upgrading your Besu client applies the network upgrade.
+For the Ethereum mainnet and public testnets (for example, Rinkeby) the milestone block definitions
+are in Hyperledger Besu. Upgrading your Besu client applies the network upgrade.
 
-For private networks, protocol upgrades must be agreed on and the network upgrade co-ordinated with all network participants. 
-The [milestone block](../Reference/Config-Items.md#milestone-blocks) at which to apply the protocol upgrade
-[is specified in the genesis file](../HowTo/Upgrade/Upgrade-Protocol.md).
+For private networks, all network participants must agree on and coordinate protocol and network
```

Comment:
Now this has been reworded I think use either _protocol upgrades_ or _network upgrades_ here rather than _protocol and network upgrades_. I think the updated wording makes it sound like there's two different sorts of upgrades rather than a network upgrade being the point in time a protocol upgrade occurs.

Reply:
Clarified. I hope.